airbyte-integrations/connectors/source-mailchimp/Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM airbyte/integration-base-python:dev
+FROM python:3.7.9-alpine3.12
 
 # Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+#RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
```

Comment:
is this intentional? (i'm guessing it's just to make your iteration cycle faster)